grakn-engine/src/main/java/ai/grakn/engine/GraknEngineConfig.java

```diff
@@ -237,20 +237,24 @@ public Properties getProperties() {
     }
 
     public String getProperty(String property) {
-        return prop.getProperty(property);
+         if(prop.containsKey(property)) {
```

Comment:
This bothered me as well :( After the `:` side I tried to put the throw, but java doesn't let you do that.